blockchain/state/iterator.go

```diff
@@ -197,6 +197,137 @@ func (it *NodeIterator) retrieve() bool {
 	return true
 }
 
+// CheckStateConsistencyParallel checks parallelly the consistency of all state/storage trie of given two state database.
+func CheckStateConsistencyParallel(oldDB Database, newDB Database, root common.Hash, quit chan struct{}) error {
+	// Create and iterate a state trie rooted in a sub-node
+	oldState, err := New(root, oldDB)
+	if err != nil {
+		return err
+	}
+
+	_, err = New(root, newDB)
+	if err != nil {
+		return err
+	}
+
+	children, err := oldState.db.TrieDB().NodeChildren(root)
+	if err != nil {
+		logger.Error("CheckStateConsistencyParallel is stop by err", "err", err)
+	}
+
+	logger.Info("CheckStateConsistencyParallel is started", "root", root.String(), "len(children)", len(children))
+
+	resultHashCh := make(chan common.Hash, 10000)
```

Comment:
```suggestion
	resultHashCh := make(chan struct{}, 10000)
```
Is `common.Hash` used in somewhere?

Reply:
Reflected